proposals/NNNN-Change_ATF_test_reports_folder_structure.md

```diff
@@ -0,0 +1,55 @@
+# Change ATF test reports folder structure
+
+* Proposal: [SDL-NNNN](NNNN-Change_ATF_test_reports_folder_structure.md)
+* Author: [Irina Getmanets](https://github.com/GetmanetsIrina)
+* Status: **Awaiting review**
+* Impacted Platforms: ATF
+
+## Introduction
+
+For now ATF reports folder has inconvenient nested struct. ATF creates struct of folders from relative path to script, e.g. runned script fom test_scripts/API/Navidation/Subscriptions/.
```

Comment:
@GetmanetsIrina What is `struct`? Please use full words instead of abbreviations.